tests/vmi_lifecycle_test.go

```diff
@@ -1205,11 +1205,17 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 				By("adding a node-feature-discovery CPU model label to a node")
 				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				featureToRequire := supportedFeatures[0]
+
+				if featureToRequire == "fpu" {
+					featureToRequire = supportedFeatures[1]
+				}
```

Comment:
Does the flakiness comes from the ordering of the CPU features ?... Or is it possible for this `fpu` feature to not be available ?

Reply:
IIUC (not sure at all) the reasoning is that it isn't safe to remove "fpu" from the cirros VM being used.
That's why I [suggested](https://github.com/kubevirt/kubevirt/pull/8202#issuecomment-1198079801) to use a disk-less VM. Then, I guess we don't really care about which features are enabled or disabled.

Reply:
... forced to say I do not have the will power to try to understand this test at all - checking for the presence of <two or more whatever random CPU features> is too weird for me.